### ISSUE TITLE
Disable the gnuplot startup message

### ISSUE THREAD

**Issue by mtreca** [emacsorphanage/gnuplot#58]
When exporting an org document containing multiple gnuplot blocks, messages of the type:
"gnuplot-mode 0.7-beta (gnuplot 3.7) -- report bugs as issues at https://github.com/emacsorphanage/gnuplot/" are spamming the echo area and message buffers.

I honestly don't think this startup message is necessary at all (removing it might also fix a version number issue present in another issue, IIRC). If having the message is deemed important, I would suggest echoing it only once when the package and loaded, and suppress subsequent invocations.

I will try to push a fix soon, if that's OK with you @conao3

**Comment by conao3** [comment]
I think it is good too!

**Comment by mtreca** [comment]
Commented out in #59
